recipes/sos-r/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "sos-r" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ed48f4f32386a9b6a7e2c13056ce43d6ff151dc33afea5a6b96316b3af0df43b
+
+build:
+  noarch: python
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ R }} -e "library('IRkernel');IRkernel::installspec()"     # [not win]
+    - {{ R }} -e \"library('IRkernel');IRkernel::installspec()\"  # [win]
+
+requirements:
+  host:
+    - ipython
+    - notebook >=5.0.0
+    - pip
+    - python >=3.6
+    - setuptools
+    - r-base
```

Comment:
Is r-base required here? That would result into pinning the R version, which does not make so much sense if the package does not contain compiled R code...